.github/workflows/update-ic.yml

```diff
@@ -84,11 +84,11 @@ jobs:
           title: 'Update IC Candid Files'
```

Comment:
Nice, thx! Should we also remove "IC" from the title?

Reply:
done!